### PR TITLE
chore(refactor): consolidate quarkus-jvm and springboot

### PR DESCRIPTION
**Pull request by lance**
While each of these runtimes needs its own buildpack, they don't need to
each have their own stack and builder images. They can share a single 'jvm'
stack and builder.

Signed-off-by: Lance Ball <lball@redhat.com>

### CODE REVIEW

File: builders/jvm/builder.toml [hunk 1]
```diff
@@ -3,14 +3,22 @@
 id = "dev.boson.quarkus-jvm"
 image = "quay.io/boson/faas-quarkus-jvm-bp:{{VERSION}}"
 
+[[buildpacks]]
```
Comment (matejvasek) [inline builders/jvm/builder.toml:6]: Wouldn't it be possible to merge those two BPs?

Comment (matejvasek) [reply builders/jvm/builder.toml:6]: probably not easily, nevermind

**Comment by matejvasek** [comment]
@lance this will new repository `quay.io/boson/faas-jvm-builder` in quay.io, right?
Won't it cause troubles like python did last time.

Maybe create the repo in quay.io in WebUI and make it public.

File: buildpacks/quarkus-jvm/buildpack.toml [hunk 2]
```diff
@@ -6,4 +6,4 @@ version = "0.0.1"
 name = "Quarkus Function Buildpack"
```
Comment (matejvasek) [inline buildpacks/quarkus-jvm/buildpack.toml:6]: we are merging builders, not buildpacks right?

Comment (matejvasek) [reply buildpacks/quarkus-jvm/buildpack.toml:6]: ah we are merging also stacks, I see now

**Comment by matejvasek** [comment]
lgtm

**Comment by lance** [comment]
> @lance this will create new repository `quay.io/boson/faas-jvm-builder` in quay.io, right?
> Won't it cause troubles like python did last time?
> 
> Maybe create the repo in advance in quay.io via WebUI and make it public.

Good suggestion! Done. https://quay.io/repository/boson/faas-jvm-builder

**Review by zroubalik — APPROVED** [review]

**Review by lkingland — APPROVED** [review]

**Pull request merged**